src/actions/archivePathReferencedThought.ts

```diff
@@ -0,0 +1,43 @@
+import _ from 'lodash'
+import Path from '../@types/Path'
+import State from '../@types/State'
+import Thunk from '../@types/Thunk'
+import { registerActionMetadata } from '../util/actionMetadata.registry'
+import reducerFlow from '../util/reducerFlow'
+import { getPathReferencePath } from './../selectors/resolvePathReference'
+import archiveThought from './archiveThought'
+
+/** Archives a thought that is referenced by a =path. */
+const archivePathReferenceThought = (state: State, options: { path?: Path; pathNote?: Path }): State => {
```

Comment:
I'm afraid my initial intuition about this was correct: Deleting the `=path` along with the Note is a bad idea. This is creating a lot of extra complexity for no real benefit. We need to keep in mind that the end goal is `=children/=path` functionality. The purpose of splitting this into separate issues was to provide an incremental on-ramp to the full functionality. Building exceptions which will need further exception in the next iteration is not going to serve us well.

I only have limited foresight architecting from afar. The real insight occurs when you're in the code and seeing how the structure is emerging. To this end, I would have expected you to notice this red flag and bring it up with me before going too deep into the implementation.